cmd/hyperfed/main.go

```diff
@@ -79,9 +82,13 @@ func NewHyperFedCommand() (*cobra.Command, []func() *cobra.Command) {
 	controller := func() *cobra.Command { return app.NewControllerManagerCommand() }
 	kubefedctlCmd := func() *cobra.Command { return kubefedctl.NewKubeFedCtlCommand(os.Stdout) }
 
+	stopCh := genericapiserver.SetupSignalHandler()
```

Comment:
Since each of these anonymous functions are called by hyperfed to establish the subcommand, I've added the `SetupSignalHandler` to hyperfed and passed it into the controller and webhook command functions.